internal/service/alarms/internal/db/repo/alarms_repository.go

```diff
@@ -111,14 +113,14 @@ func (ar *AlarmsRepository) GetAlarmSubscription(ctx context.Context, id uuid.UU
 }
 
 // UpsertAlarmEventRecord insert and updating an AlarmEventRecord.
-func (ar *AlarmsRepository) UpsertAlarmEventRecord(ctx context.Context, records []models.AlarmEventRecord) error {
+func (ar *AlarmsRepository) UpsertAlarmEventRecord(ctx context.Context, records []models.AlarmEventRecord, globalCloudId uuid.UUID) error {
```

Comment:
Wouldn't it be cleaner to add the globalCloudID when the `AlarmEventRecord` is built rather than deferring this step to the repository/db code to manage?

Reply:
yeah it was originally like that. Will revert

Reply:
There other option is to defer it all the way out to the converter to add it just before it is sent out as either a notification or an API response.

Reply:
> defer it all the way out to the converter to add it just before it is sent out 

Yeah going with that